src/clients/contact-client.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const Client = require('./base-client')
+const uploadFileBrowser = require('../browser/upload-file')
```

Comment:
No, the contact client should work for browser application but also for Nodejs applications (AWS Lambda). That's why we use a proxy => see [asset client](https://github.com/infopark-customers/bima-shark-js-sdk/blob/develop/src/clients/asset-client.js#L4)

Reply:
I suspected it should work in this way, but did not find any references before and suspect my VS did not want to search inside `node_modules` folder. Thank you for hint.